Lib/argparse.py

```diff
@@ -521,19 +521,16 @@ def _format_action(self, action):
 
         # no help; start on same line and add a final newline
         if not action.help:
-            tup = self._current_indent, '', action_header
-            action_header = '%*s%s\n' % tup
+            action_header = f'{"":{self._current_indent}}{action_header}'
 
         # short action name; start on the same line and pad two spaces
         elif len(action_header) <= action_width:
-            tup = self._current_indent, '', action_width, action_header
-            action_header = '%*s%-*s  ' % tup
+            action_header = f'{"":{self._current_indent}}{action_header:{action_width}}  '
```

Comment:
```suggestion
            action_header = f'{"":{self._current_indent}}{action_header:<{action_width}}  '
```

Reply:
Did my suggestion pass the 80 char line width limit?

Reply:
Hey @sunmy2019, excellent catch!
Because the old behaviour would align left and my f-string was aligning right, and no tests failed, does this mean I should add a test for this case?

(The line was already longer than 80 chars, it wasn't your catch that made it go over. Should I also break it apart?)

Thanks!

Reply:
> does this mean I should add a test for this case?

I think it might not be necessary unless it is stated in the doc

Reply:
I've fixed the alignment, thanks.

What are the next steps?

Reply:
Wait for someone with write permission. But argparse has been lack of maintainers, so no one is responsible here.

Reply:
Alright, cheers.